### PR TITLE
Fixes units for rpc data sizes to correctly be bytes rather than MB

### PR DESCRIPTION
**Pull request by dinooliva**

### CODE REVIEW

File: core/src/main/java/com/google/instrumentation/stats/RpcConstants.java [hunk 1]
```diff
@@ -251,15 +251,15 @@
   public static final IntervalViewDescriptor RPC_CLIENT_REQUEST_BYTES_INTERVAL_VIEW =
       IntervalViewDescriptor.create(
           "rpc client request_bytes",
-          "Minute and Hour stats for request in MBs",
+          "Minute and Hour stats for request in bytes",
```
Comment (sebright) [inline core/src/main/java/com/google/instrumentation/stats/RpcConstants.java]: I think this should be "request size in bytes".

File: core/src/main/java/com/google/instrumentation/stats/RpcConstants.java [hunk 2]
```diff
@@ -251,15 +251,15 @@
...
           RPC_CLIENT_REQUEST_BYTES,
           IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
           Arrays.asList(RPC_CLIENT_METHOD));
 
   public static final IntervalViewDescriptor RPC_CLIENT_RESPONSE_BYTES_INTERVAL_VIEW =
       IntervalViewDescriptor.create(
           "rpc client response_bytes",
-          "Minute and Hour stats for response size MBs",
+          "Minute and Hour stats for response size bytes",
```
Comment (sebright) [inline core/src/main/java/com/google/instrumentation/stats/RpcConstants.java]: Should this be "in bytes"?

File: core/src/main/java/com/google/instrumentation/stats/RpcConstants.java (hunk 1 above)
Comment (dinooliva) [inline core/src/main/java/com/google/instrumentation/stats/RpcConstants.java]: Done.

File: core/src/main/java/com/google/instrumentation/stats/RpcConstants.java (hunk 2 above)
Comment (dinooliva) [inline core/src/main/java/com/google/instrumentation/stats/RpcConstants.java]: Done.

**Pull request merged**